x-pack/plugin/async/build.gradle

```diff
@@ -25,5 +22,7 @@ dependencyLicenses {
   ignoreSha 'x-pack-core'
 }
 
+//no tests
+test.enabled = false
```

Comment:
Can we similarly use the lazy configuration apis here, ie. `tasks.named("test").configure { }`.

Reply:
done https://github.com/elastic/elasticsearch/pull/60554/commits/b541ac89d257eca864a0ce89b174e21fa3918612